src/main/java/seedu/address/logic/parser/DeleteEventCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
```

Comment:
Try to be explicit: Change DeleteCommand to DeleteEventCommand as the former refers to the delete person command.

Reply:
Thanks. Have made the changes.